### PR TITLE
Adds psychium, a drink that gives you psychotic brawling

### PR DESCRIPTION
**Pull request by enthethan**
:cl: enthethan
:add: psychium, gives you enraged drunken father energy to beat your wife and son into a coma
/:cl:

### CODE REVIEW

**Comment by karmaisblackandbluepilled** [comment]
damn nigga you fucked up that changelog and shit

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 1]
```diff
@@ -436,3 +436,25 @@
 		M.emote(pick("twitch","laugh","frown"))
 	..()
 	. = 1
+
+/datum/reagent/drug/psychium
+	name = "Psychium"
+	description = "A drink derivative of the infamous drug psycho,popular with wastelanders."
```
Comment (ghost) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm]: typo

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 2]
```diff
@@ -436,3 +436,25 @@
...
+	reagent_state = LIQUID
+	color = "#FAFAFA"
+	taste_description = "Pure rage"
```
Comment (ghost) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm]: lower case

**Comment by ghost** [comment]
Add your changelog and it'll be a +1 from me

**Review by enthethan — COMMENTED** [review]
unga?

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm (hunk 1 above)
Comment (ghost) [reply code/modules/reagents/chemistry/reagents/drug_reagents.dm]: still a typo

**Comment by spessbro** [comment]
trash first pr holy shit -1

**Comment by enthethan** [comment]
> trash first pr holy shit -1

t-thanks

**Comment by enthethan** [comment]
hmm, not compiling without errors for travis or whatever, but it is for me

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm (hunk 1 above)
Comment (yoyobatty) [reply code/modules/reagents/chemistry/reagents/drug_reagents.dm]: put a space after the comma please

**Comment by enthethan** [comment]
merge it or i will call git admsin!!! and they will ban you !

**Comment by karmaisblackandbluepilled** [comment]
you need 2 designer +1's
also +1

**Comment by Manly-Dwarf** [comment]
fucked that changelog UP

**Comment by Toxici11i** [comment]
+1 just because i want to beat my wife HARDER

**Comment by ghost** [comment]
+1, waiting for headmins to merge

**Review by ghost — APPROVED** [review]
nice

**Pull request merged**